src/mousetracker.js

```diff
@@ -2570,12 +2574,12 @@
         if ( eventInfo.stopPropagation ) {
             $.stopEvent( event );
         }
-        if ( eventInfo.shouldCapture && !implicitlyCaptured ) {
-           //$.console.log('pointerdown calling capturePointer() ' + (tracker.userData ? tracker.userData.toString() : '') + ' ' + (event.target === tracker.element ? 'tracker.element' : ''));
-           capturePointer( tracker, gPoint );
-        } else if ( !eventInfo.shouldCapture && implicitlyCaptured ) {
-           //$.console.log('pointerdown calling releasePointer() ' + (tracker.userData ? tracker.userData.toString() : '') + ' ' + (event.target === tracker.element ? 'tracker.element' : ''));
-           releasePointer( tracker, gPoint ); //TODO should we do this? Investigate when implementing bubble handling
```

Comment:
@iangilman  Here's the main bit that fixes the issue, as described in the text I added to this PR's description